src/AppInstallerCommonCore/Manifest/MsixManifestValidation.cpp

```diff
@@ -36,6 +36,22 @@ namespace AppInstaller::Manifest
         return errors;
     }
 
+    MsixManifestValidation::~MsixManifestValidation()
+    {
+        AICLI_LOG(Core, Info, << "Removing downloaded installers");
+        for (const auto& installerPath : m_downloadedInstallers)
+        {
+            try
+            {
+                std::filesystem::remove(installerPath);
+            }
+            catch (...)
+            {
+                AICLI_LOG(Core, Warning, << "Failed to remove downloaded installer: " << installerPath.u8string());
```

Comment:
```suggestion
                AICLI_LOG(Core, Warning, << "Failed to remove downloaded installer: " << installerPath);
```

We have overload for std::filesystem::path already